### ISSUE TITLE
Last word of the line and first word of the next line are joined when using NewRichTextFromMarkdown

### ISSUE THREAD

**Issue by lugu** [fyne-io/fyne#2647]
### Describe the bug:
RichText widget resulting from parse markdown join the last word of a line with the subsequent first word of the next line.

### To Reproduce:
Create a rich text widget from a markdown text which has several lines. 

### Screenshots:
![markdown-demo](https://user-images.githubusercontent.com/354646/141860044-2b0ffdac-52ee-4359-ae31-04263f3f25b3.png)

### Example code:
```
package main

import (
        "fyne.io/fyne/v2/app"
        "fyne.io/fyne/v2/widget"
)

func main() {
        win := app.New().NewWindow("Markdown Demo")

        md := `
first line
second line
`
        content := widget.NewRichTextFromMarkdown(md)
        win.SetContent(content)
        win.ShowAndRun()
}
```

### Device (please complete the following information):
 - **OS:** Linux
 - **Version:** ?
  - **Go version:** 1.5
 - **Fyne version:** 2.1

**Comment by andydotxyz** [comment]
Thanks, fixed in the upcoming v2.1.2 - you can test using `release/v2.1.x` branch

**Comment by lugu** [comment]
Works for me. Thank you!
